scripts/bsvpreprocess.py

```diff
@@ -113,7 +113,7 @@ def nexttok(s):
             else:
                 defs[sym] = ''
         elif tok == 'include':
-            m = re.search('"?([-_A-Za-z0-9.]+)"?', s)
+            m = re.search('"+([-_A-Za-z0-9.]+)"+', s)
```

Comment:
I reverted this change, will submit again when I ran into an example of this issue.